src/components/dialogs/input-hooks.js

```diff
@@ -454,7 +454,7 @@ const DELAY = 1000;
 export const useSearchEquipmentField = ({ handleOpenSearchDialog, label }) => {
```

Comment:
strange naming. Seems like it's a very generic custom hook. Maybe `useButton` is a better name. Then change `handleOpenSearchDialog` by `handleClick` maybe ?

Reply:
done